bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -408,46 +412,45 @@ spec:
       - CREATE
       - UPDATE
       resources:
-      - instrumentations
+      - opentelemetrycollectors
     sideEffects: None
     targetPort: 9443
     type: MutatingAdmissionWebhook
-    webhookPath: /mutate-opentelemetry-io-v1alpha1-instrumentation
+    webhookPath: /mutate-opentelemetry-io-v1alpha1-opentelemetrycollector
   - admissionReviewVersions:
     - v1
-    - v1beta1
     containerPort: 443
     deploymentName: opentelemetry-operator-controller-manager
-    failurePolicy: Fail
-    generateName: mopentelemetrycollector.kb.io
+    failurePolicy: Ignore
+    generateName: mpod.kb.io
     rules:
     - apiGroups:
-      - opentelemetry.io
+      - ""
       apiVersions:
-      - v1alpha1
+      - v1
       operations:
       - CREATE
       - UPDATE
       resources:
-      - opentelemetrycollectors
+      - pods
     sideEffects: None
     targetPort: 9443
     type: MutatingAdmissionWebhook
-    webhookPath: /mutate-opentelemetry-io-v1alpha1-opentelemetrycollector
+    webhookPath: /mutate-v1-pod
   - admissionReviewVersions:
     - v1
-    - v1beta1
     containerPort: 443
     deploymentName: opentelemetry-operator-controller-manager
-    failurePolicy: Ignore
-    generateName: vinstrumentationdelete.kb.io
+    failurePolicy: Fail
```

Comment:
I see that this is not a change in behavior, but we might want to reconsider in a follow up issue/PR to change this to `Ignore`. As it is, we'd block people from creating pods if the operator has a bug or isn't available.

Reply:
Yes this is not related to this PR, the webhook for the collector use fail policy https://github.com/open-telemetry/opentelemetry-operator/blob/abe7d5358997f9cb0a16f6115b652fc47e51b3c6/apis/v1alpha1/opentelemetrycollector_webhook.go#L37.

Reply:
Might still be worth reconsidering in a follow-up PR/issue.